source/fundamentals/builders/sort.txt

```diff
@@ -71,86 +58,91 @@ sample collection, ``sort_example``, that contains the following documents:
    {"_id": 2, "letter": "a", "food": "donuts and coffee"}
    {"_id": 6, "letter": "c", "food": "maple donut"}
 
+The Sorts Class
+---------------
 
-Sorting Direction
------------------
+The ``Sorts`` class is a builder that provides static factory methods for all sort
+criteria operators supported by MongoDB. These methods return a ``Bson`` object
+that you can pass to the  ``sort()`` method of a ``FindIterable`` instance or to 
+``Aggregates.sort()``. If you want to learn more about ``Aggregates``, see our 
+:doc:`guide on the Aggregates builder </fundamentals/builders/aggregates>`.
 
-The ``Sorts`` class provides methods for specifying the direction of your sort.
-The direction of your sort can either be **ascending** or **descending**.
-An ascending sort orders your results from smallest to largest. A
-descending sort orders your results from largest to smallest.
+The following two code snippets are equivalent, however we recommend you use the
+``Sorts`` builder to specify your sort criteria:
 
-Here are some examples of data sorted in ascending order:
+.. code-block:: java
 
-* Numbers: 1, 2, 3, 43, 43, 55, 120
-* Dates: 1990-03-10, 1995-01-01, 2005-10-30, 2005-12-21 
-* Words (ASCII): Banana, Dill, carrot, cucumber, hummus
+   import static com.mongodb.client.model.Sorts.ascending;
+
+   // <MongoCollection setup code here>
+   
+   //Recommended
+   collection.find().sort(ascending("_id"));
 
-Here are some examples of data sorted in descending order:
+.. code-block:: java
 
-* Numbers: 100, 30, 12, 12, 9, 3, 1
-* Dates: 2020-01-01, 1998-12-11, 1998-12-10, 1975-07-22 
-* Words (reverse ASCII): pear, grapes, apple, Cheese
+   // <MongoCollection setup code here>
+   
+   //Not Recommended
+   collection.find().sort(new Document("_id", 1));
+
+The code snippets above both sort the documents in the
```

Comment:
The "both" here feels a bit redundant. You could also say "Both code snippets..."
```suggestion
The code snippets above sort the documents in the
```

Reply:
Thank you for the note.  I agree this phrasing could be improved. think I prefer "Both code snippets", as in my head it stresses equivalence more heavily.

```suggestion
Both code snippets above sort the documents in the
```